app/controllers/boxes_controller.rb

```diff
@@ -17,6 +17,7 @@ def index
   def show
     return unless authorize_resource(@box, READ_BOX)
     @can_delete = has_access?(@box, DELETE_BOX)
+    @can_blind = @box.box_transfers.count == 0
```

Comment:
We have a method for that (it's a little faster as it won't even run a COUNT in SQL):

```suggestion
    @can_blind = !@box.transferred?
```